src/content/docs/workflows/about-workflows/index.mdx

```diff
@@ -0,0 +1,44 @@
+---
+page_id: 85c135db-f55a-4edf-8d1b-0af4140c1925
+title: About workflows
+sidebar:
+  order: 1
+relatedArticles:
+  - d672fca7-6c6a-49f8-85c9-6e86ce99f440
+  - 8554d648-fc2f-4ef9-aaa9-4f27b3c05205
+  - d360c480-49a8-47d3-b178-1abd34ae669f
+---
+
+In Kinde, workflows exist to give you (almost) unlimited power over how Kinde works for you. 
+
+Develop code in your own repo, and push it to a Kinde workflow, where we execute it in ways that you define, triggered by Kinde events. 
+
+Currently, workflows are limited to the following triggers:
+
+- `user:post_authentication` fires after the user has completed single factor authentication (e.g. email + password or Google), and before authorization.
+- `m2m:token_generation` - an m2m transaction fires.
+- `user:new_password_provided` - a user creates or resets a password.
+- `user:existing_password_provided` - a user signs in with their password.
```

Comment:
The user hasn’t signed in, they have requested to sign in and given their current password